### PR TITLE
persist incoming time as time for better comparison

### PR DESCRIPTION
**Pull request by saikumar9**
* [Pod of War - 188415677](https://www.pivotaltracker.com/story/show/188415677)

### CODE REVIEW

**Review by jacobkagon — APPROVED** [review]
Change looks good. We're now converting the updated at string to time in CreatedOrUpdatedAtProcessor and persisting after_updated_at as Time instead of DateTime.

File: app/domain/operations/families/create.rb [hunk 1]
```diff
@@ -5,21 +5,34 @@
 module Operations
   module Families
     # This class will create a Family object
+    # @example
+    #   create_operation = Operations::Families::Create.new
+    #   result = create_operation.call(params)
```
Comment (jayreddy519) [inline app/domain/operations/families/create.rb:10]: thanks for adding this.

File: app/domain/operations/families/created_or_updated_processor.rb [hunk 2]
```diff
@@ -43,24 +44,35 @@ def create_response_transmittable(comparison_entity, request_objects)
...
         Failure("Invalid After Updated At timestamp. Error raised #{e.message}")
       end
 
+      # Transforms the timestamp string to a Time object.
+      #
+      # @param after_updated_at_str [String] the timestamp string
+      # @return [Dry::Monads::Result] the result of the transformation
+      def transform_timestamp(after_updated_at_str)
+        ::AcaEntities::Operations::DateTimeTransforms::ConvertStringToTime.new.call(
+          { time_string: after_updated_at_str }
```
Comment (jayreddy519) [inline app/domain/operations/families/created_or_updated_processor.rb:68]: calling operation to convert time_string to time.

File: spec/domain/operations/families/create_spec.rb [hunk 3]
```diff
@@ -37,6 +41,12 @@
       it "returns a family" do
         expect(@result.success).to be_a(Family)
       end
+
+      it 'sets inbound_after_updated_at for family as expected' do
+        expect(
+          @result.success.inbound_after_updated_at.to_time
+        ).to eql(after_save_updated_at.to_time)
+      end
```
Comment (jayreddy519) [inline spec/domain/operations/families/create_spec.rb:49]: Added specs for the change.

**Review by jayreddy519 — APPROVED** [review]
LGTM.

**Pull request merged**